yarn.lock

```diff
@@ -294,40 +294,40 @@
     "@types/yargs" "^13.0.0"
 
 "@polkadot/api-contract@^0.97.0-beta.39":
-  version "0.97.0-beta.39"
-  resolved "https://registry.yarnpkg.com/@polkadot/api-contract/-/api-contract-0.97.0-beta.39.tgz#3ac80bb39f3592e65fc5cd305b92db8d4ce71b28"
-  integrity sha512-O3p29g5M8Z3eGgdyPlPoAu57SVRocW1VeUUSfgLySS6q9+N1fttaVEzEwbAdktsQFUdxt06sFyYgiCWdZ0ZVew==
+  version "0.97.1"
```

Comment:
update.sh just took the next stable version after the @beta version we were using before.

When using the beta-versions, you need to update them manually in the package.json

It's currently `0.99.0-beta.13` for `api` and `api-contract`

Reply:
That will possibly resolve the `PhantomData` data error